### ISSUE TITLE
代码存在内存泄漏问题，建议进行优化

### ISSUE THREAD

**Issue by lehaifeng000** [bubbliiiing/faster-rcnn-pytorch#35]
loss累加的时候直接相加，而没有+=loss.item()，导致在某些时候资源没有得到释放，内存不断增加
代码修改成下面这样正常
total_loss += total.item() 
rpn_loc_loss += rpn_loc.item() 
rpn_cls_loss += rpn_cls.item() 
roi_loc_loss += roi_loc.item() 
roi_cls_loss += roi_cls.item()
